xbmc/utils/FileUtils.h

```diff
@@ -10,6 +10,7 @@
 
 #include <memory>
 #include <string>
+#include "FileItem.h"
```

Comment:
No need for this include. remove it.
Instead, add `#include <memory>`- for `std::shared_ptr`